### ISSUE TITLE
Slurm integration python2/3 confusion

### ISSUE THREAD

**Issue by jabl** [Mellanox/ufm_sdk_3.0#35]
The slurm integration scripts in `plugins/SLURM-Integration` are launched via the `ufm-{prolog,epilog}.sh` scripts which launch them by running `python3`. However the python scripts themselves contain the shebang path `/usr/bin/python`, which depending on the system setup, can still point to the old python 2.x interpreter. Can you please fix so that python 2.x is not used by accident e.g. when testing?

```
ufm_sdk_3.0/plugins/SLURM-Integration$ git grep usr/bin/python
ufm_slurm_base.py:#!/usr/bin/python
ufm_slurm_epilog.py:#!/usr/bin/python
ufm_slurm_prolog.py:#!/usr/bin/python
ufm_slurm_utils.py:#!/usr/bin/python
```

**Comment by flx42** [comment]
I agree with the assessment of @jabl, can we fix this please? In addition, python files that are not meant to be executed directly should probably not be executable (like `ufm_slurm_utils.py`), nor have a shebang at all (`ufm_slurm_base.py` and `ufm_slurm_utils.py`).

@anasbadaha @mike-dubman
